vignettes/how-to/use-rhino-on-windows.Rmd

```diff
@@ -0,0 +1,12 @@
+---
+title: "How-to: Use Rhino on Windows"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How-to: Use Rhino on Windows}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+You can use JavaScript & Sass tools on Windows 10.
+You will need to [install Node.js and yarn](https://go.appsilon.com/rhino-system-dependencies)
+and enable [Developer Mode](https://consumer.huawei.com/en/support/content/en-us15594140/).
```

Comment:
Huawei? can you find some official link, like something from Microsoft's documentation? i think it'd be better

Reply:
Done, makes sense. Huawei's page is better though (IMO) :laughing: